tests/framework/types.go

```diff
@@ -39,12 +39,12 @@ type VirtualMachine struct {
 
 func (vm VirtualMachine) Create() (string, string, error) {
 	args := []string{"create", "-f", vm.Manifest}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtOcPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace,"oc", args...)
```

Comment:
Use the path to oc and virtctl are path of run parameters.

Reply:
I've copied solution from my another PR